ignite/contrib/handlers/stores.py

```diff
@@ -45,8 +45,15 @@ def update(self, engine: Engine) -> None:
         output = self.output_transform(engine.state.output)
         self.data.append(output)
 
-    def attach(self, engine: Engine) -> None:
+    def store(self, engine: Engine) -> None:
```

Comment:
@radekosmulski I think the docstring should be update to show this new feature.

Reply:
sorry, completely missed that, will add! :slightly_smiling_face: